### PR TITLE
Various small fixes

### PR DESCRIPTION
**Pull request by gsmet**
Noticed while working on quarkus-github-bot and https://github.com/quarkusio/quarkus/issues/31355

### CODE REVIEW

File: independent-projects/bootstrap/runner/src/main/java/io/quarkus/bootstrap/runner/Timing.java [hunk 1]
```diff
@@ -97,15 +97,19 @@ public static void printStartupTime(String name, String version, String quarkusV
         //Use a BigDecimal so we can render in seconds with 3 digits precision, as requested:
         final BigDecimal secondsRepresentation = convertToBigDecimalSeconds(bootTimeNanoSeconds);
         String safeAppName = (name == null || name.trim().isEmpty()) ? UNSET_VALUE : name;
-        String safeAppVersion = (version == null || version.trim().isEmpty()) ? UNSET_VALUE : version;
+        String safeAppVersion = (version == null || version.trim().isEmpty() || version == "null") ? UNSET_VALUE : version;
```
Comment (gastaldi) [inline independent-projects/bootstrap/runner/src/main/java/io/quarkus/bootstrap/runner/Timing.java:100]:
```suggestion
        String safeAppVersion = (version == null || version.trim().isEmpty() || version.equals("null")) ? UNSET_VALUE : version;
```

Comment (gsmet) [reply independent-projects/bootstrap/runner/src/main/java/io/quarkus/bootstrap/runner/Timing.java:100]: Outch, what the hell...

Comment (gsmet) [reply independent-projects/bootstrap/runner/src/main/java/io/quarkus/bootstrap/runner/Timing.java]: Funnily enough, I suppose the string was interned as it was working.

I think I'm too used to write `thing == null` without quotes :).

Thanks for catching it.

**Comment by gastaldi** [comment]
Test failures are unrelated

**Review by gastaldi — APPROVED** [review]

**Pull request merged**